### PR TITLE
BZ 1499032 Added note on latest glusterfs package

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1499032

### CODE REVIEW

**Comment by mburke5678** [comment]
@eparis We got a request to add the minimum required version of glusterfs that is needed to install OpenShift. The version in subscription-manager (glusterfs-*-3.7.9) is apparently wrong for 3.6. 
Is this in your purview?

**Comment by jarrpa** [comment]
@mburke5678 I can start looking into this. I'll comment on the BZ.

**Review by jarrpa — CHANGES_REQUESTED** [review]
I think this should do towards the end of the document, something like:

```
If the package was already installed, make sure the latest version is installed:

----
# yum update glusterfs-fuse
----
```

**Comment by mburke5678** [comment]
@jarrpa I added your sentence and command at the end of the topic. Moving the BZ to `On_QA`.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -857,12 +857,20 @@ enabled:
...
-pods which will use GlusterFS volumes. For RPM-based systems, the
+The `mount.glusterfs` command must be available on all nodes that will host
+pods that will use GlusterFS volumes. For RPM-based systems, the
 *glusterfs-fuse* package must be installed:
 
 ----
 # yum install glusterfs-fuse
 ----
+
+If GlusterFS is already installed on the nodes, make sure the latest version is installed:
```
Comment (ahardin-rh) [inline install_config/install/prerequisites.adoc]: s/make sure the/ensure that the

**Comment by ahardin-rh** [comment]
@mburke5678 Just one suggestion from me. Otherwise, looks good!

**Comment by mburke5678** [comment]
[rev_history]
|xref:../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Installation and Configuration -> Prerequisites]
|Added note to ensure the lastest version of Glusterfs is installed.
%

**Pull request merged**